### PR TITLE
Refactor optimetrics

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: examples/00-EDB/00_EDB_Create_VIA.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 ###############################################################################
 # Import the EDB Layout Object
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-# This example imports the EDB layout object and initializes it on version 2021.2.
+# This example imports the EDB layout object and initializes it on version 2022R2.
```
Comment (MaxJPRey) [inline examples/00-EDB/00_EDB_Create_VIA.py:11]: @maxcapodi78 Bellow we are mentioning 2022.1 in `edb()` initializer.
```suggestion
# This example imports the EDB layout object and initializes it on version 2022R1.
```

File: examples/00-EDB/01_edb_example.py [hunk 2]
```diff
@@ -35,11 +35,11 @@
 # Launch EDB
 # ~~~~~~~~~~
 # This example launches the :class:`pyaedt.Edb` class.
-# This example uses EDB 2021.2 and uses SI units.
+# This example uses EDB 2022R2 and uses SI units.
```
Comment (MaxJPRey) [inline examples/00-EDB/01_edb_example.py:38]:
```suggestion
# This example uses EDB 2022R1 and uses SI units.
```

File: examples/00-EDB/03_5G_antenna_example.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 ###############################################################################
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # This example imports the `Hfss3dlayout` object and initializes it on version
-# 2021.2.
+# 2022R2.
```
Comment (MaxJPRey) [inline examples/00-EDB/03_5G_antenna_example.py:11]:
```suggestion
# 2022R1.
```

File: examples/00-EDB/05_Plot_nets.py [hunk 4]
```diff
@@ -31,9 +31,9 @@
 # Launch EDB
 # ~~~~~~~~~~
 # This example launches the :class:`pyaedt.Edb` class.
-# This example uses EDB 2021.2 and uses SI units.
+# This example uses EDB 2022R2 and uses SI units.
```
Comment (MaxJPRey) [inline examples/00-EDB/05_Plot_nets.py:34]:
```suggestion
# This example uses EDB 2022R1 and uses SI units.
```

File: examples/01-Modeling-Setup/HFSS_CoordinateSystem.py [hunk 5]
```diff
@@ -21,10 +21,10 @@
 ###############################################################################
 # Launch AEDT in Graphical Mode
 # -----------------------------
-# This example launches AEDT 2021.2 in graphical mode.
+# This example launches AEDT 2022R2 in graphical mode.
```
Comment (MaxJPRey) [inline examples/01-Modeling-Setup/HFSS_CoordinateSystem.py:24]:
```suggestion
# This example launches AEDT 2022R1 in graphical mode.
```

File: examples/02-HFSS/Advanced_Far_Field.py [hunk 6]
```diff
@@ -39,9 +39,9 @@
 ###############################################################################
 # Launch AEDT in Non-Graphical Mode
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-# This example launches AEDT 2021.2 in graphical mode.
+# This example launches AEDT 2022R2 in graphical mode.
```
Comment (MaxJPRey) [inline examples/02-HFSS/Advanced_Far_Field.py:42]:
```suggestion
# This example launches AEDT 2022R1 in graphical mode.
```

File: pyaedt/application/Design.py [hunk 7]
```diff
@@ -1042,6 +1038,90 @@ def desktop_install_dir(self):
...
+        ----------
+        aedt_object : object
+            AEDT Object on which search for property. It can be any oProperty (ex. oDesign).
+        object_name : str
+            Path to the object list. Example `"DesginName\Boundaries"`.
+
+        Returns
+        -------
+        object
+            Aedt Object if Any.
```
Comment (MaxJPRey) [inline pyaedt/application/Design.py:1076]:
```suggestion
            Aedt Object if any.
```

**Review by MaxJPRey — APPROVED** [review]

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

**Pull request merged**